src/ctap/crypto_wrapper.rs

```diff
@@ -101,7 +101,9 @@ pub fn aes256_cbc_decrypt(
 // We shouldn't compare private keys in prod without constant-time operations.
 #[cfg_attr(test, derive(PartialEq, Eq))]
 pub enum PrivateKey {
-    Ecdsa(ecdsa::SecKey),
+    // We store the seed instead of the key since we can't get the seed back from the key. We could
+    // store both if we believe deriving the key is done more than once and costly.
+    Ecdsa([u8; 32]),
```

Comment:
Not sure I like the design now. Some of the variants are actual private keys, while others are not. And since it's an enum, this detail is not hidden from the outside. It would probably still work, but it makes it a bit more error-prone to do something like, say, cache the actual ECDSA private key. But I don't have a good idea how to do it better, so I guess it's okay?

Reply:
What they had in common before was that they implemented `sign` I think, but I agree to your statement. In the end, the purpose `PrivateKey` is to have something you can simply call `sign` on, and store it etc. And storing seeds should be the implementation detail.

I agree that's not related to this PR. Maybe if `PrivateKey` becomes more complicated, it will eventually need to be better encapsulated.